letsencrypt/client/standalone_authenticator.py

```diff
@@ -266,30 +266,23 @@ def already_listening(self, port):  # pylint: disable=no-self-use
         If so, also tell the user via a display notification.
 
         .. warning::
-            The current implementation is Linux-specific.  (On other
-            operating systems, it will simply not detect bound ports.)
-            This function can only usefully be run as root.
+            On some operating systems, this function can only usefully be
+            run as root.
 
         :param int port: The TCP port in question.
         :returns: True or False."""
 
+        listeners = [conn.pid for conn in psutil.net_connections() \
+                     if conn.status == 'LISTEN' and \
+                     conn.type == socket.SOCK_STREAM and \
+                     (conn.laddr == ('0.0.0.0', port) or \
```

Comment:
What about connections that bind to specific network interfaces rather than `0.0.0.0`/`::`? Wouldn't `conn.laddr[0] == port` be enough?